### PR TITLE
add a notification drawer like trello

### PR DESCRIPTION
**Pull request by jtbairdsr**
Referencing wekan/wekan#2471.
@xet7 here is what I have figured out for adding a notification drawer...

as mentioned in the issue this does not include a scheduled task to clean up old notifications but I am working on that.

---
This change is [Reviewable](https://reviewable.io/reviews/wekan/wekan/2975)

### CODE REVIEW

File: client/components/notifications/notificationsDrawer.jade [hunk 1]
```diff
@@ -0,0 +1,16 @@
+template(name='notificationsDrawer')
+  section#notifications-drawer(class="{{#if $.Session.get 'showReadNotifications'}}show-read{{/if}}")
+    .header
+      if $.Session.get 'showReadNotifications'
+        a.toggle-read Filter by Unread
```
Comment (xet7) [inline client/components/notifications/notificationsDrawer.jade:5]: Please change this to:
```
a.toggle-read {{_ 'filter-by-unread'}}
```

File: client/components/notifications/notificationsDrawer.jade [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      else
+        a.toggle-read View All
```
Comment (xet7) [inline client/components/notifications/notificationsDrawer.jade:7]: Please change this to:
```
a.toggle-read {{_ 'view-all'}}
```

File: client/components/notifications/notificationsDrawer.jade [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      h5 Notifications
```
Comment (xet7) [inline client/components/notifications/notificationsDrawer.jade:8]: Please change this to:
```
h5 {{_ 'notifications'}}
```

File: i18n/en.i18n.json [hunk 4]
```diff
@@ -64,7 +64,7 @@
   "activity-unchecked-item": "unchecked %s in checklist %s of %s",
   "activity-checklist-added": "added checklist to %s",
   "activity-checklist-removed": "removed a checklist from %s",
-  "activity-checklist-completed": "completed checklist __checklist__ at card __card__ at list __list__ at swimlane __swimlane__ at board __board__",
+  "activity-checklist-completed": "completed checklist %s of %s",
```
Comment (xet7) [inline i18n/en.i18n.json:67]: Please change bottom of this file to:
```
  "newUserPopup-title": "New User",
  "notifications": "Notifications",
  "view-all": "View All",
  "filter-by-unread": "Filter by Unread",
  "mark-all-as-read": "Mark all as read"
```

Comment (xet7) [reply i18n/en.i18n.json:67]: I did add these translation strings already to Wekan, so that they will be translated.

**Comment by jtbairdsr** [comment]
@xet7 

I updated the static text in the notification drawer to use i18n... I am trying to duplicate the error now...

**Comment by jtbairdsr** [comment]
@xet7 

I think that should fix your error...

Thanks again for all that you do!

**Comment by john9838** [comment]
Hello, 

I do not find that file in my folders. 
How do I access this file client/components/notifications/notificationsDrawer.jade as well as to the "client" folder?? 

It's so weird I do not find it anywhere.

John

**Comment by xet7** [comment]
@john9838 

```
git clone https://github.com/wekan/wekan
```
https://github.com/wekan/wekan/blob/master/client/components/notifications/notificationsDrawer.jade

**Comment by john9838** [comment]
@xet7 Thank you so much for your answer. 
I didn't realize I could do that. You are amazing. Thanks.

**Comment by xet7** [comment]
@john9838 

Here is how to build WeKan code: https://github.com/wekan/wekan/wiki/Emoji

**Comment by john9838** [comment]
@xet7 I will definitely need it. We'll be working on it. Wekan is fantastic. Thank you so much for your contributions. We all benefits from it. Thanks.

**Comment by xet7** [comment]
@john9838 

Thanks! You can send pull requests for any feature/fix/etc you like to be added to WeKan.

**Pull request merged**